public/components.css

```diff
@@ -0,0 +1,599 @@
+    background-color: rgba(255, 255, 255, 0.8);
+    text-align: center;
+    padding: 20px 0;
+    font-size: 30px;
+}
+
+.product-img {
+    grid-area: "product-img";
+    background: url("../images/item1.jpeg");
+    background-repeat: no-repeat;
+    height: 100%;
+    background-position: center;
+    width: 100%;
+    margin-top: 1rem;
+    background-size: contain;
+}
+
+.product-regulator {
+    margin-top: 1rem;
+    margin-left: 20px;
+    text-align: center;
+    grid-area: "regulator";
+    width: 70%
+}
+
+.product-desc {
+    grid-area: "desc";
+    margin-top: 1rem;
+}
+
+.product-card {
+    display: grid;
+    background: white;
+    grid-template-columns: 190px 1fr;
+    grid-template-rows: 10rem 3rem;
+    grid-template-areas: "product-img desc" "regulator desc";
+    grid-gap: 10px;
+    margin: auto;
+    border: 1px solid black;
+    max-width: 80%;
+    padding: 1rem 0;
+    border-radius: 20px;
+}
+
+.product-regulator span {
+    border: 1px solid black;
+    margin: 0 4px;
+    padding: 3px 17px;
+    font-weight: bold;
+}
+
+.product-regulator button {
+    padding: 0 10px;
+    border-radius: 50%;
+    height: 30px;
+    border: 1px solid gray;
+    background-color: white;
+    font-weight: bold;
+    width: 30px;
+}
+
+.product-information h2,
+.product-information h3 {
+    display: inline;
+    margin-right: 1rem;
+    margin-top: 1rem;
+}
+
+.product-information * {
+    margin-bottom: 20px;
+}
+
+.cart-option {
+    width: 200px;
+    text-align: left;
+}
+
+.cart-opt-img>img {
+    width: 90%;
+    background-size: contain;
+}
+
+.cart-opt-img {
+    position: relative;
+}
+
+.cart-opt {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+}
+
+.cart-tag {
+    display: inline-block;
+    position: absolute;
+    top: -10%;
+    left: -10%;
+}
+
+.rectangular-image {
+    border: 1px solid white;
+    height: 500px;
+    width: 500px;
+    padding: 1rem;
+    box-shadow: 0 0 1rem 0 rgba(0, 0, 0, .2);
+}
+
+.circular-image {
+    border: 1px solid white;
+    border-radius: 50%;
+    height: 500px;
+    width: 500px;
+    padding: 1rem;
+    box-shadow: 0 0 1rem 0 rgba(0, 0, 0, .2);
+}
+
+.nav-content {
+    border: 1px solid white;
+    margin-left: 2rem;
+    display: inline;
+    align-self: center;
+    font-weight: bolder;
+    padding: 0.6rem;
+    border-radius: 1rem;
+}
+
+.nav-content:hover {
+    background-color: black;
+    color: white;
+    box-shadow: 9px 8px 2rem 0px rgb(255 255 255 / 20%);
+    cursor: pointer;
+}
+
+.lists {
+    display: flex;
+    flex-direction: column;
+    border: 1px solid white;
+    width: 70%;
+    padding: 1rem 1rem;
+    justify-content: center;
+    margin: auto;
+    box-shadow: 0 0 1rem 0 rgba(0, 0, 0, .2);
+    border-radius: 1rem;
+}
+
+.list-items {
+    border: 1px solid black;
+    justify-content: space-around;
+    text-align: center;
+    padding: 1rem;
+    color: black;
+    font-weight: bold;
+}
+
+form {
```

Comment:
give class names to forms , label ,and input tags

Reply:
Done